pandas/tests/frame/test_cumulative.py

```diff
@@ -7,6 +7,9 @@
 """
 
 import numpy as np
+import pytest
+
```

Comment:
can use the _test_decorator version of this

Reply:
are you referring to pandas/util/_test_decorators.py?

we have nothing for np-dev there (just skip_if_np_lt) . also those are skips and don't we want an xfail?

Reply:
ok instead of importing from here (this is not meant for this)
import from pd. where all of the lives (you will to add the import in pandas/__init__.py

Reply:
sure. we currently have a mixture of imports for _np_version_under1p* and wasn't sure which was the preferred.